lib/nexmo_developer/app/views/layouts/documentation.html.erb

```diff
@@ -17,18 +17,30 @@
         </nav>
       </div>
     </div>
-    <main class="Vlt-main Nxd-main">
-      <%= render partial: 'layouts/partials/notices' %>
-      <div class="Vlt-grid">
-        <div class="Vlt-col Vlt-col--3of4 Nxd-tutorial">
-          <%= yield %>
-          <%= render partial: 'layouts/partials/improve_this_page' %>
+    <% if @not_show_nav%>
+      <main class="Vlt-main Nxd-main">
+        <%= render partial: 'layouts/partials/notices' %>
+        <div class="Vlt-grid">
+          <div class="Vlt-col Vlt-col--5of4 Nxd-tutorial">
+            <%= yield %>
+            <%= render partial: 'layouts/partials/improve_this_page' %>
+          </div>
         </div>
-        <div class="Vlt-col Vlt-col--1of4 Vlt-sidenav Nxd-tutorial-sidenav">
-            <%= navigation_from_content(content: @content) %>
-        </div>
-       </div>
-    </main>
+      </main>
+    <%else%>
+      <main class="Vlt-main Nxd-main">
+        <%= render partial: 'layouts/partials/notices' %>
+        <div class="Vlt-grid">
+          <div class="Vlt-col Vlt-col--3of4 Nxd-tutorial">
+            <%= yield %>
+            <%= render partial: 'layouts/partials/improve_this_page' %>
+          </div>
+            <div class="Vlt-col Vlt-col--1of4 Vlt-sidenav Nxd-tutorial-sidenav">
```

Comment:
wrap this in if/else using `controller_name` rails helper and remove @not_show_nav